package.json

```diff
@@ -1,57 +1,52 @@
 {
-  "name": "brainsandbeards-com",
-  "description": "Brains & Beards website",
-  "version": "0.2.0",
-  "author": "Brains & Beards",
-  "dependencies": {
-    "expo": "^37.0.12",
-    "expo-av": "^8.1.0",
-    "gatsby": "^2.22.15",
-    "gatsby-image": "^2.3.1",
-    "gatsby-plugin-anchor-links": "^1.1.1",
-    "gatsby-plugin-client-side-redirect": "^1.1.0",
-    "gatsby-plugin-favicon": "3.1.6",
-    "gatsby-plugin-feed": "^2.4.1",
-    "gatsby-plugin-mailchimp": "^5.1.2",
-    "gatsby-plugin-matomo": "^0.8.3",
-    "gatsby-plugin-next-seo": "^1.5.0",
-    "gatsby-plugin-react-helmet": "^3.2.1",
-    "gatsby-plugin-react-native-web": "^3.1.0",
-    "gatsby-plugin-sass": "^2.3.3",
-    "gatsby-plugin-sharp": "^2.6.9",
-    "gatsby-plugin-sitemap": "^2.3.1",
-    "gatsby-remark-better-embed-video": "^2.4.0",
-    "gatsby-remark-copy-linked-files": "^2.2.1",
-    "gatsby-remark-images": "^3.2.1",
-    "gatsby-remark-prismjs": "^3.4.1",
-    "gatsby-remark-responsive-iframe": "^2.3.1",
-    "gatsby-source-filesystem": "^2.2.2",
-    "gatsby-transformer-remark": "^2.7.1",
-    "gatsby-transformer-sharp": "^2.5.3",
-    "prismjs": "^1.19.0",
-    "react": "^16.13.1",
-    "react-dom": "^16.8.6",
-    "react-helmet-async": "^1.0.6",
-    "react-id-swiper": "^1.6.8",
-    "react-native": "^0.62.2",
-    "react-native-animated-code-input": "./react-native-animated-code-input-1.0.0.tgz",
-    "react-native-web": "~0.11.7",
-    "sass": "^1.54.9"
-  },
+  "name": "brains-beards-site",
+  "version": "1.0.0",
+  "private": true,
+  "description": "brains-beards-site",
+  "author": "Wojciech Ogrodowczyk",
   "keywords": [
-    "gatsby",
-    "gatsby-starter",
-    "starter"
+    "gatsby"
   ],
-  "license": "MIT",
   "scripts": {
-    "build": "gatsby build",
     "develop": "gatsby develop",
-    "format": "prettier --write 'src/**/*.js'"
+    "start": "gatsby develop",
+    "build": "gatsby build",
+    "serve": "gatsby serve",
+    "clean": "gatsby clean",
+    "typecheck": "tsc --noEmit"
+  },
+  "dependencies": {
+    "@mdx-js/react": "^2.3.0",
+    "gatsby": "^5.7.0",
```

Comment:
The newest is now 5.9.0 🌝

Reply:
I'll take a break from updating Gatsby and leave it as-is :)